source/_posts/2017-07-22-robot-framework-as-embedded-systems-validation-framework.markdown

```diff
@@ -0,0 +1,350 @@
+---
+author: Piotr Król
+layout: post
+title: "Robot Framework for embedded systems validation - iPXE automation use case"
+date: 2017-07-22 22:42:44 +0200
+comments: true
+categories: apu2 python robotframework validation automation
+---
+
+Recently we started to prepare to [ECC2017](https://ecc2017.coreboot.org/)
+conference. One of topics that we considered was a system for development and
+validation automation.
+
+As maintainers of PC Engines platforms in coreboot we debug and fix quite a lot
+of bugs, but to take full responsibility for our code everything should be
+validated each time we do release. Limited resources lead us to automation and
+as Python enthusiasts we decided to evaluate Robot Framework as first candidate.
+
+When preparing to mentioned conference I found that we lack PXE server from
+which I could run or install needed OSes. Also there was no ready to use
```

Comment:
@acihy please rethink and try to read "there was no ready-to-use configuration". IMO it is correct.